api/types/types.proto

```diff
@@ -1271,6 +1277,13 @@ message RoleOptions {
     // RequireSessionMFA specifies whether a user is required to do an MFA
     // check for every session.
     bool RequireSessionMFA = 13 [ (gogoproto.jsontag) = "require_session_mfa,omitempty" ];
+
+    // Lock specifies the locking mode (strict|best_effort) to be applied with
+    // the role.
+    string Lock = 14 [
```

Comment:
Why not name this `LockingMode`?

Reply:
Because that's how it turned out to be specified in the RFD after several iterations of naming discussions.